main.py

```diff
@@ -171,33 +256,44 @@
+                    member = main_guild.get_member(user)
+
+                    if member == None:
+                        await message.author.send("It doesn't seem like <@{}> is part of the server! I am sorry, I can't add them to your **conspiracy channel**.".format(user))
+                    elif db.isParticipant(user,False,True) == True:
+                        if int(db_get(user,'abducted')) == 1:
+                            abductees.append(member)
+                        elif int(db_get(user,'frozen')) == 1:
+                            frozones.append(member)
+                        elif db.isParticipant(user,False,False) == False:
+                            deadies.append(member)
                         else:
-                            sorry = await message.channel.send("It doesn't seem like <@{}> is part of this server! I am sorry, I can't add them to your channel.".format(member))
-                            temp_msg.append(sorry)
+                            viewers.append(member)
+                    else:
+                        deadies.append(member)
 
                 intro_msg = creation_messages.cc_intro([v.id for v in viewers])
 
                 # Role objects (based on ID)
                 roles = main_guild.roles # Roles from the guild
                 game_master_role = discord.utils.find(lambda r: r.id == game_master, roles)
-                dead_participant_role = discord.utils.find(lambda r: r.id == dead_participant, roles)
-                frozen_participant_role = discord.utils.find(lambda r: r.id == frozen_participant, roles)
                 default_permissions = {
-                    main_guild.default_role: discord.PermissionOverwrite(read_messages=False),
-                    frozen_participant_role: discord.PermissionOverwrite(send_messages=False),
-                    dead_participant_role: discord.PermissionOverwrite(read_messages=True, send_messages=False),
-                    game_master_role: discord.PermissionOverwrite(read_messages=True),
+                    main_guild.default_role: discord.PermissionOverwrite(read_messages=False,send_messages=False),
+                    game_master_role: discord.PermissionOverwrite(read_messages=True,send_messages=True),
                     client.user: discord.PermissionOverwrite(read_messages=True,send_messages=True),
                     **{
-                        member: discord.PermissionOverwrite(read_messages=True) for member in viewers
+                        member: discord.PermissionOverwrite(read_messages=True,send_messages=True) for member in viewers
+                    },
+                    **{
+                        member: discord.PermissionOverwrite(read_messages=True,send_messages=False) for member in frozones
                     },
+                    **{
+                        member: discord.PermissionOverwrite(read_messages=True,send_messages=False) for member in deadies
+                    }
```

Comment:
I assume this has been tested?

Reply:
Unsure. I vaguely remember having checked it in a channel's info bar, but it always failed because my test subjects had an admin role.